onyo/lib/commands.py

```diff
@@ -254,48 +245,40 @@ def _edit_asset(inventory: Inventory,
                 asset: dict | UserDict,
                 operation: Callable,
                 editor: str | None) -> dict | UserDict:
-    r"""Edit `asset` via configured editor and a temporary asset file.
+    r"""Edit an ``asset`` (as a temporary file) with ``editor``.
+
+    A helper for ``onyo_edit()`` and ``onyo_new(edit=True)``.
 
-    Utility function for `onyo_edit` and `onyo_new(edit=True)`.
-    This is editing a temporary file initialized with `asset`. Once
-    the editor is done, `asset` is updated from the file content and
-    `operation` is tried in order to validate the content for a
-    particular purpose (Currently used: Either `Inventory.add_asset`
-    or `Inventory.modify_asset`).
-    User is asked to either keep editing or accept the changes
-    (if valid).
+    The asset content is edited as a temporary file. Once the editor is done,
```

Comment:
I find this explanation sounds incomplete. It mentions temporary files that get edited, and that one can "accept" after being presented the diff with the promt. This as a whole makes it sound as if one accepts changes for the temporary file.

It should either mention:
1. the changes are applied to the original file
2. fully explain the four options, e.g. "skip" vs. "abort" as an option is not clear.

Nowhere it says that multiple assets can be edited at once, which makes these options even more unclear.

Reply:
I have added a sentence detailing how the temporary file works.

However, the rest of the comments don't really apply, as `_edit_asset()` is a helper function, and is non-user-facing. And because it's a private function, it's not even included in RTD rendering. It's documented to for developer convenience.

> Nowhere it says that multiple assets can be edited at once, 

Because this function only accepts one asset at a time.